app/models/work_version.rb

```diff
@@ -98,6 +100,16 @@ class WorkVersion < ApplicationRecord
     end
   end
 
+  # 6/3/2022 : Added to prevent https://app.honeybadger.io/projects/77112/faults/85827019
```

Comment:
Honeybadger URLs expire, so I was hoping you'd link somewhere on the web that describes the "String contains null bytes" problem.